documentation/builders/audio.md

```diff
@@ -63,7 +63,9 @@ Bluetooth setup consists of three steps
 
 To pair and connect, follow these steps. This will be a one-time setup.
 
-```bash
+```
+sudo bluetoothctl
+bash
```

Comment:
I think there is a small mistake here..

it should be

```
```bash
sudo bluetoothctl
...
```